### ISSUE TITLE
MOOSE 4.1.0 not installing correctly when installed from github

### ISSUE THREAD

**Issue by pgleeson** [OpenSourceBrain/MOOSEShowcase#8]
@subhacom I see the version of Moose is updated to 4.1 on master. However the test we have for building this:
https://github.com/OpenSourceBrain/MOOSEShowcase/blob/development/.github/workflows/build-ci.yml
is currently failing:
https://github.com/OpenSourceBrain/MOOSEShowcase/actions/runs/9078258468/job/24944905681#step:5:941
```
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/opt/hostedtoolcache/Python/3.9.19/x64/lib/python3.9/site-packages/moose/__init__.py", line 21, in <module>
    import moose._moose as _moose
ModuleNotFoundError: No module named 'moose._moose'
```
Any thoughts?

**Comment by subhacom** [comment]
This should be fixed with pull request [#478](https://github.com/BhallaLab/moose-core/pull/478)

**Comment by pgleeson** [comment]
Thanks @subhacom! Works now. 
Note I had to install a few more dependencies to get it to compile:
https://github.com/OpenSourceBrain/MOOSEShowcase/pull/9/files#diff-0e17310d2ef9b97f7a3b1f8ab8ee6b34af5b2b9f3e309214e64fd7e4620dd468
